### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by arybolovlev**
### Description

This PR add issue templates to the repo.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ - Kubernetes version:
+
+### YAML Manifest File
+```yaml
+# Copy-paste your YAML manifest here
+```
+
+### Output Log
+<!---
+Please provider a link to a GitHub Gist containing the complete output log.
```
Comment (alexsomesan) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]:
```suggestion
Please provide a link to a GitHub Gist containing the complete output log.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Please DO NOT paste the debug output in the issue; just paste a link to the Gist.
+--->
+
+### Kubectl Outputs
+<!---
+Please provider a link to a GitHub Gist containing the complete kubectl outputs.
```
Comment (alexsomesan) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]:
```suggestion
Please provide a link to a GitHub Gist containing the complete kubectl outputs.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+# Copy-paste your YAML manifest here
+```
+
+### Output Log
+<!---
+Please provider a link to a GitHub Gist containing the complete output log.
+Please DO NOT paste the debug output in the issue; just paste a link to the Gist.
+--->
+
+### Kubectl Outputs
```
Comment (alexsomesan) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: What is this asking for exactly? Which kind of outputs? Console output of kubectl commands? The use of plural (outputs) is confusing me a bit.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+<!---
+Please provider a link to a GitHub Gist containing the complete kubectl outputs.
+Please DO NOT paste the kubectl output in the issue; just paste a link to the Gist.
+--->
+
+### Steps To Reproduce
+Steps to reproduce the behavior:
```
Comment (alexsomesan) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]: Is this an example of the actual recommended steps required of the user to perform?
Let's mark as "example" if that is the case.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
+name: 🚀 Feature request
+about: Suggest an idea or an improvement for this project
+title: '🚀 '
+labels: 'feature-request'
+assignees: ''
+
+---
+<!---
+Please DO NOT remove any fields from this template. If there is nothing to add, leave N/A.
```
Comment (alexsomesan) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]:
```suggestion
Please DO NOT remove any fields from this template. If there is nothing to add, fill in N/A.
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
+---
+name: 🤔 Question
+about: If you need help figuring something out
+title: '🤔 '
+labels: 'question'
+assignees: ''
+
+---
+<!---
+Please DO NOT remove any fields from this template. If there is nothing to add, leave N/A.
```
Comment (alexsomesan) [inline .github/ISSUE_TEMPLATE/question.md:10]:
```suggestion
Please DO NOT remove any fields from this template. If there is nothing to add, fill in N/A.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
+---
+name: 🐛 Bug report
+about: Create a report to help us improve
+title: '🐛 '
+labels: 'bug'
+assignees: ''
+
+---
+<!---
+Please DO NOT remove any fields from this template. If there is nothing to add, leave N/A.
```
Comment (alexsomesan) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]:
```suggestion
Please DO NOT remove any fields from this template. If there is nothing to add, fill in N/A.
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+--->
+### Operator Version, Kind and Kubernetes Version
+ - Operator version:
+ - Kind: Workspace | Module
+ - Kubernetes version:
+
+### YAML Manifest File
+<!---
+Please let us know if you have a question about YAML manifest.
```
Comment (alexsomesan) [inline .github/ISSUE_TEMPLATE/question.md:19]:
```suggestion
Please let us know if you have a question about a specific YAML manifest.
```

**Review by alexsomesan — COMMENTED** [review]
I spotted some autocorrect bloopers and other minor phrasing observations.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (arybolovlev) [reply .github/ISSUE_TEMPLATE/bug_report.md:28]: This is for `kubectl` outputs that are related to the reported issue. For instance: `kubectl get module`, `kubectl get workspace`, `kubectl get secret`, `kubectl get configmap`, and the same but `kubectl describe`, instead of `kubectl get`.
An artificial example here can be incorrect storage for the outputs, like "It says 'sensitive', but it appears in the configmap". 🤨 In this case, great to see `get workspace | configmap | secret`.

Comment (alexsomesan) [reply .github/ISSUE_TEMPLATE/bug_report.md:28]: Got it. May I suggest to rephrase this to "Output of relevant `kubectl` commands" ?

**Review by alexsomesan — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (arybolovlev) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yes sir! Done. :) That makes more sense, thanks!
